MidwestAccessCoalition/lockbox_rails
# Add support request nav section—issue 121

## Link to issue:  
Fixes issue #121 

## Changelog
### Assets
* add styling for SR nav bar

### Models & Controllers
* add easy lookup at SRs via `has_many :support_requests` on `lockbox_partner` model
* add `@support_requests` ivar for use in view

### Views
* add SR nav bar to `lockbox_partners/support_requests#show` page
* fix unbalanced div tags; add some new lines for readability

## Steps for QA/Special Notes:
* must login as lockbox partner rather than normal user, e.g. `user: fluffy@catsclinic.com, password: heytherefancypants4321` (see also [db/seeds.rb](https://github.com/MidwestAccessCoalition/lockbox_rails/blob/master/db/seeds.rb) for more examples)

## Relevant Screenshots: 
### Support Request nav bar in context:
<img width="1280" alt="Screen Shot 2019-09-29 at 6 14 27 PM" src="https://user-images.githubusercontent.com/6441226/65841161-4be1fb00-e2e5-11e9-8917-d644dcbb78ce.png">

### With Older link disabled:
<img width="912" alt="Screen Shot 2019-09-29 at 6 14 46 PM" src="https://user-images.githubusercontent.com/6441226/65841174-57cdbd00-e2e5-11e9-96a3-17c6c4fa1445.png">

### With Newer link disabled:
<img width="911" alt="Screen Shot 2019-09-29 at 6 14 53 PM" src="https://user-images.githubusercontent.com/6441226/65841178-61572500-e2e5-11e9-9fb8-cbce30583ca3.png">

### TODO
